Core/GDCore/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -68,6 +68,10 @@ gd::String EventsCodeGenerator::GenerateRelationalOperatorCall(
     }
   }
 
+  if (relationalOperator == "startsWith") {

```

Comment:
Is this code causing any issue?
I guess it could be modified to check for quote directly as we can no longer rely on the string length.
I don't know if this code is still relevant or not.
```JS
  if (relationalOperator.size() > 2)
    relationalOperator = relationalOperator.substr(
        1,
        relationalOperator.length() - 1 -
            1);  // Relational operator contains quote which must be removed.
```

Reply:
I checked that code previously and is not afecting, because that code only is in charge of removing the double quote (") from the start and beginning, and affects all operators I tried from the string operator field.